curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5efada803cbd2bbdab94e332.md

```diff
@@ -42,10 +42,14 @@ The third image should have a `src` attribute set to `https://cdn.freecodecamp.o
 
 ```js
 const catsImg = document.querySelectorAll('figure > img')[1];
-assert(
-  catsImg &&
-    catsImg.getAttribute('src').toLowerCase() === 'https://cdn.freecodecamp.org/curriculum/cat-photo-app/cats.jpg'
-);
+assert.equal(catsImg?.getAttribute('src')?.trim().toLowerCase(), 'https://cdn.freecodecamp.org/curriculum/cat-photo-app/cats.jpg');
+```
+
+The `src` attribute must not contain any extra spaces before or after the link. Remove the extra spaces.
```

Comment:
```suggestion
The `src` attribute should not contain any extra spaces before or after the URL. Remove the extra spaces.
```

I still do not think it is correct of us to test for this as an error. The image will load just fine with spaces, so it is valid code.

If we do test for it, the language should be changed, "must not" much stronger implies it _can't_ have spaces, but "should not" is just us saying we don't want it to have spaces (for whatever reason).

Personally, I think we should just drop this test. I don't see why we have to test for spaces when it is valid code, and we already trimmed in the previous test so that works with or without spaces.